Havit.Blazor.Components.Web.Bootstrap/wwwroot/HxSearchBox.js

```diff
@@ -17,7 +17,14 @@
 function handleKeyDown(event) {
 	const key = event.key;
 
-	event.target.hxSearchBoxDotnetObjectReference.invokeMethodAsync("HxSearchBox_HandleInputKeyDown", key);
+	// Only call Blazor for keys that need server-side handling (navigation keys)
+	// Regular typing characters should be handled locally to prevent character dropping
+	const navigationKeys = ["Enter", "NumpadEnter", "ArrowUp", "ArrowDown"];
+	const needsServerHandling = navigationKeys.includes(key) || event.target.hxSearchBoxKeysToPreventDefault.includes(key);
```

Comment:
Rename to `needsBlazorHandling`

Reply:
Do not combine with `hxSearchBoxKeysToPreventDefault`, these do not influence the need for Blazor handling.